Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/NodePaletteModel.h

```diff
@@ -59,6 +61,17 @@ namespace ScriptCanvasEditor
         AZStd::string m_tooltip;
     };
 
+    struct RegisterSmallOperatorNodeInformation
```

Comment:
This all can just be the DataDrivenTypeInfo, and doesn't need to refer to small operators